packages/runtime/container-runtime/src/summaryGenerator.ts

```diff
@@ -389,7 +389,10 @@ export class SummaryGenerator {
                 duration: broadcastDuration,
                 referenceSequenceNumber: summarizeOp.referenceSequenceNumber,
                 summarySequenceNumber: summarizeOp.sequenceNumber,
-                handle: summarizeOp.contents.handle,
+                handle: {
+                    value: summarizeOp.contents.handle,
```

Comment:
This one seemed to potentially be sensitive data as well

Reply:
handles are produces by storage, so they cannot be coming from code (or user).
They are either random numbers or checksums (like sha1), so it's safe to have them untagged.